docs/source/css/extra.css

```diff
@@ -13,7 +13,6 @@ p {
 .md-typeset table td,
 .md-typeset table th {
   font-size: 0.7rem;
-  line-height: 0.5;
```

Comment:
```suggestion
  padding: .5em 1.25em !important;
```

Reply:
Yes we could do this but it's not selective. But if @hoechenberger is okay with affecting all tables I'm happy to commit